Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -484,7 +484,7 @@ public SiteExtensionManager(IContinuousJobsManager continuousJobManager, ITrigge
                                 Executable exe = externalCommandFactory.BuildCommandExecutable(installScript,
                                     installationDirectory,
                                     _settings.GetCommandIdleTimeout(), NullLogger.Instance);
-                                exe.ExecuteWithProgressWriter(NullLogger.Instance, _traceFactory.GetTracer(), String.Empty);
+                                exe.ExecuteWithProgressWriter(NullLogger.Instance, _traceFactory.GetTracer(), installationArgs ?? String.Empty);
```

Comment:
I squashed/merged your initial change. Make sure you rebase any additional change over the new master